force-app/main/default/classes/HH_Container_LCTRL.cls

```diff
@@ -38,7 +38,15 @@ public with sharing class HH_Container_LCTRL {
 
     /** @description our exception object for Field Level & Object Security errors. */
     private class permsException extends Exception {}
-    
+
+    private static Boolean canReadContactUndeliverable = 
```

Comment:
Don't we need to add namespace for the field name on these calls to canRead?

Reply:
Since it's using a direct reference to the Field Describe, I think it's fine! This is how Daniel did it on the Contact Merge page, no issues there.

Reply:
Apex is smart enough to handle the namespace dynamically 99.99% of the time.